tests/test_form.py

```diff
@@ -123,6 +123,27 @@ def test_choose_submit_from_selector(value):
     assert res.status_code == 200 and res.text == 'Success!'
 
 
+@pytest.mark.parametrize("value", [
+    pytest.param('continue', id='first'),
+    pytest.param('cancel', id='second'),
+])
+def test_choose_submit_from_selector_image(value):
+    """Test choose_submit by passing a CSS selector argument."""
```

Comment:
We're all guilty of writing docstrings that are not sufficiently detailed, but for the sake of this as a learning experience, I'll add that it would be nice if you could make the docstring say more specifically what the test is designed to do (you can even reference an Issue number here if you like).

Reply:
Take a look.

Reply:
Looks great, thanks for the extra detail!